src/models/cost_functions/ImportExportCost.jl

```diff
@@ -9,7 +9,7 @@ An operating cost for imports/exports and ancillary services from neighboring ar
 employs a CostCurve{PiecewiseIncrementalCurve} with an implied zero cost at zero power.
 """
 mutable struct ImportExportCost <: OperationalCost
-    "Buy Price Curves data to import power, which can be a time series of [`PiecewiseStepData`](@extref) or a
+    "Buy Price Curves data to import power, which can be a time series of [`PiecewiseStepData`] or a
```

Comment:
Why was this `@extref` removed? There's a IS docstring for `PiecewiseStepData`, and it's linked with DocumenterInterLinks
```suggestion
    "Buy Price Curves data to import power, which can be a time series of [`PiecewiseStepData`](@extref) or a
```

Reply:
It was giving me an error of not found... I'll add it again